app/keepers/keepers.go

```diff
@@ -442,7 +442,7 @@ func (appKeepers *AppKeepers) WireICS20PreWasmKeeper(
 	bApp *baseapp.BaseApp,
 	hooksKeeper *ibchookskeeper.Keeper) {
 	// Setup the ICS4Wrapper used by the hooks middleware
-	wasmHooks := ibchooks.NewWasmHooks(hooksKeeper, nil) // The contract keeper needs to be set later
+	wasmHooks := ibchooks.NewWasmHooks(hooksKeeper, nil, "osmo") // The contract keeper needs to be set later
```

Comment:
Its silly, but the way to get the constant from SDK properly is:
```
sdk.GetConfig().GetBech32AccountAddrPrefix()
```

Reply:
ahh this is really useful! I was looking into how to do this! (I thought the sdk constant would be updated, but that just gives me "stake")

Reply:
> sdk.GetConfig().GetBech32AccountAddrPrefix()

Will this work on any chain? If so, we can remove the param and get it inside the function, but maybe it's good to have it configurable anyway?

Reply:
this _should_ work on any chain.

I don't have a preference between arg or computed locally